packages/react-cookie/src/Cookie.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```

Comment:
This follows the universal provider pattern so closely, would it make sense to call this `CookiesUniversalProvider`?